docs/linelist.rst

```diff
@@ -0,0 +1,121 @@
+.. highlight:: rest
+
+.. _LineList:
+
+******************
+LineList Class
+******************
+
+.. index:: LineList
+
+Overview
+========
+
+This Class is designed to organize and handle information about atomic
+and/or molecular transition lines (e.g. HI Lya, CIV 1548, Hydrogen
+Balmer series, etc.) observed in a variety of astrophysical
+environments. It is currently implemented for absorption lines, but we
+expect to also include common emission lines in the near future.
+
+..
+   (:ref:`AbsLine Class`).  add this back in when written
+
+By definition, a LineList is a unique collection of transitions lines
+specified by a predefined keyword (or list of such keywords). At the
+moment, we have implemented six keywords:
+
+* 'ISM' : "All" ISM lines (can be overwhelming!)
+* 'Strong' : Strong ISM lines (most common absorption line transitions observed)
+* 'HI' : HI Lyman series
+* 'H2' : H2 (Lyman-Werner)
+* 'CO' : CO UV band-heads
+* 'EUV' :  Extreme UV lines
+
+We note that a subset of transitions can also be specified using the
+`subset_line()` methods (see Section XXX). In this way, one can create
```

Comment:
Need to add a reference here. If you want to add a placeholder for future docs covering subset_line, please put it in a comment (see Absline Class comment above).